### ISSUE TITLE
Unable to join our server and MV-Iventories is mentioned in the error stack

### ISSUE THREAD

**Issue by harl** [Multiverse/Multiverse-Inventories#465]
### Information

* **Server version:**  Paper 1.17.1 build 157 (Git: f521a18)

* **Full output of `/mv version -p`:**  https://j.mp/2VyW6F0

* **Server log:**  https://gist.github.com/harl/ab4a5f6532f3673104cb6602a8025182

### Help request

**Problem**
+ I seem unable to properly log onto the server, so that my connection to it gets lagged/stalled and terminated. (This is practically a 3-man server, definitely not a general issue with the server itself or its network connections.)
+ I don't know exactly what the issue is, but one of the last things I did before today was to update MultiVerse and its modules, so now that I'm seeing MV-Inventories in the console (and log files) and it seems to be somehow related to this new issue, I'm hoping someone can take a look at that log and tell me if this could be caused by MV(-Inventories) or if it's something else.
+ I'm not good at understanding those error stacks, which seems to be the key to getting to the bottom of these things.

**What I have tried**
+ I've restarted the server.
+ I'm not sure if it's alright to just cut MV and/or MVI out of the equation (as in trying to join the server without MV/MVI loaded), but I could do that if it helps to isolate the cause.  (Same goes for loading the server with only MV (+modules) enabled.)
+ (Also, I could try to manually edit userdata and move myself to another world first before joining to see if the underlying issue may be related to our main survival world or at least the chunks I'm in etc.)

**Comment by benwoo1110** [comment]
There looks to be an issue with file read/write on your server. Either some corruption or slow I/O performance. While there is mv-inv mentioned, there are also other instance of the lag happening at other task. But all the common issue with them is they are doing some file I/O task. So its currently not conclusive that mv-inv is the cause of anything here.

**Comment by harl** [comment]
Thanks for the response.

In the meantime, whatever was wrong with it seems to have fixed itself - for now. (This confuses me a little.)
There was a (scheduled) server restart and now I can join, leave and rejoin the sever just fine.

I only saw the errors that mentioned MVI in them and - before I created this issue ticket - I had observed the server for signs of general misbehaviour, but there were no obvious systemic hiccups or similar, there was nothing wrong with the files themselves, and the memory (supposedly) checked out fine too.

Anyway, I guess this issue is solved (until it resurfaces, whatever caused what was wrong).
